CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog for the Clash project
 
+## 1.8.2 *TBD*
+
+* Add `ShowX`, `NFDataX` instances for `Proxy`
+
```

Comment:
Could you follow the procedure outlined in [`changelog/README.md`](https://github.com/clash-lang/clash-compiler/blob/master/changelog/README.md)?